docs/source/conf.py

```diff
@@ -238,3 +238,13 @@
     'traits': ('http://docs.enthought.com/traits', None),
     'pyface': ('http://docs.enthought.com/pyface', None),
 }
+
+
+def autodoc_skip_member(app, what, name, obj, skip, options):
+    # Skip load_tests targeting unittest discover
+    is_load_tests = what == "module" and name == "load_tests"
```

Comment:
The `what == "module"` part was not there in the pyface PR. This additional check is more strict: it excludes class method named "load_tests" or exceptions named "load_tests"... which aren't part of unittest discover protocol.